Automattic/jetpack
# Sharing buttons style improvements

Fixes https://github.com/Automattic/jetpack/issues/35044

## Proposed changes:

Adds color controls to Sharing Buttons block, similar to Social Links in core Gutenberg.

https://github.com/Automattic/jetpack/assets/60262784/9d6da045-52b8-4b79-acfb-a452a0bd7da5


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up your site with local changes
 - Make sure you have beta block enabled. In your environment (example Docker) wp-config.php you should have define( 'JETPACK_BLOCKS_VARIATION', 'beta' );
- Add Sharing Buttons block to a post.
- Check controls
